<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <arg name="depth_image" default="/zed/zed_node/depth/depth_registered"/>
    <arg name="rgb_image" default="/zed/zed_node/left/image_rect_color"/>
    <arg name="rgb_camera_info" default="/zed/zed_node/left/camera_info"/>
    <arg name="events_topic" default="/celex5_mipi/events"/>
    <arg name="events_camera_info" default="/celex5_mipi/display/binary_img/camera_info"/>
    <arg name="store_dir" default="$(find celex5_calibration)/output/"/>

    <include file="$(find zed_wrapper)/launch/zed.launch"/>
    <include file="$(find celex5_ros)/launch/celex5_ros_node.launch"/>


    <!--
    <node pkg="celex5_calibration"
          name="event_registration_node"
          type="event_registration_node"
          output="screen"
          required="true">
        <param name="depth_image" type="string" value="$(arg depth_image)"/>
        <param name="rgb_image" type="string" value="$(arg rgb_image)"/>
        <param name="rgb_camera_info" type="string" value="$(arg rgb_camera_info)"/>
        <param name="events_topic" type="string" value="$(arg events_topic)"/>
        <param name="events_camera_info" type="string" value="$(arg events_camera_info)"/>
        <param name="store_dir" type="string" value="$(arg store_dir)"/>
    </node>
    -->

</launch>
